style.css

```diff
@@ -0,0 +1,89 @@
+.toolbar {
+  height: 100px;
+  background: #fff;
+  box-shadow: 0 1px 0 #dddfe6;
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.nav {
+  width: 109px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+  color: #1a2236;
+}
+
+.headline {
+  display: flex;
+  justify-content: left;
+}
+
+.text {
+  margin-left: 60px;
+}
+
+.main__article {
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  height: 220px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 110px;
+  color: #1a2236;
+}
+
+.bio {
```

Comment:
- [ ] kyndly use the correct fonts and colors, 

### Figma design 
![image](https://user-images.githubusercontent.com/95243996/198583375-c2fd9980-8f8a-4119-b0ab-c5952d7b4ab7.png)

### your project 

![image](https://user-images.githubusercontent.com/95243996/198583764-83b8ee55-0111-4f89-827f-22a9915bd215.png)